src/cython/vapoursynth.pyx

```diff
@@ -1225,6 +1225,46 @@ cdef FrameProps createFrameProps(RawFrame f):
 Mapping.register(FrameProps)
 
 
+cdef class ChannelLayout(int):
```

Comment:
That doesn't need to be cdef and couold be instantiated directly. Indeed, I wonder if this might be a thing, that people might want to do.

I mean it's just an int at the end of the day. It doesn't contain pointers to vapoursynth.

Reply:
I just didn't want people to use it directly, but if you want it as part of the public API, sure.